decorators/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator
+  def correct_name
+    @nameable.correct_name[0..9] if @nameable.correct_name.length > 10
```

Comment:
- [ ] Kindly add an else case for this method as now it will return `nil` if the name has 10 characters or less.